book.rb

```diff
@@ -0,0 +1,26 @@
+require './item'
+
+# Book class for book object creation.
+class Book < Item
+  attr_accessor :publisher, :cover_state
+
+  @books = []
+  def initialize(publisher:, cover_state:, **attributes)
+    super(attributes)
+    @publisher = publisher
+    @cover_state = cover_state
+    self.class.all << self
+  end
+
+  def can_be_archived?
+    Date.today.year - @publish_date.year > 10 || @cover_state == 'Bad'
+  end
```

Comment:
- [ ] In order to avoid code repetition, since `can_be_archived` had been implemented in the `superclass` (item.rb); kindly use the super keyword to access the method. 👍 This will show a good comprehension of **Inheritance** in **Ruby**. **Note: This is applicable to `game.rb` and `music_album.rb` as well.**
```suggestion
  def can_be_archived?
    super || @cover_state == 'Bad'
  end
```

Reply:
Thanks that really is helpful.
i just chabged it. 😄